pallets/cash/src/chains.rs

```diff
@@ -25,13 +25,15 @@ pub enum ChainId {
     Gate,
     Eth,
     Dot,
+    Matic, // xxx todo: due to rebrand ticker and name don't match for MATIC/Polygon.. so review this enum name is this what we want to call this?
 }
 
 impl ChainId {
     pub fn to_account(self, addr: &str) -> Result<ChainAccount, Reason> {
         match self {
             ChainId::Gate => Ok(ChainAccount::Gate(Gateway::str_to_address(addr)?)),
             ChainId::Eth => Ok(ChainAccount::Eth(Ethereum::str_to_address(addr)?)),
+            ChainId::Matic => Ok(ChainAccount::Matic(Polygon::str_to_address(addr)?)),
```

Comment:
these should be append only, if Dot has ever been written to storage, it will be wrong next time it is read after this patch. We'd probably get away with it this time, but it's a huge pain if this get's messed up